audit/projects/kubernetes-public/secrets/slack-moderator-words-config/versions.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "createTime": "2021-06-30T17:01:11.914742Z",
+    "etag": "\"15c5feab6c38f6\"",
+    "name": "projects/127754664067/secrets/slack-moderator-words-config/versions/4",
+    "replicationStatus": {
+      "automatic": {}
+    },
+    "state": "ENABLED"
+  },
```

Comment:
This was me fixing the secret: https://github.com/kubernetes/k8s.io/issues/2013#issuecomment-871585026